src/Apps/ArtworkRecommendations/artworkRecommendationsRoutes.tsx

```diff
@@ -0,0 +1,40 @@
+import loadable from "@loadable/component"
+import type { RouteProps } from "System/Router/Route"
+import { graphql } from "react-relay"
+
+const ArtworkRecommendationsApp = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "artworkRecommendationsBundle" */ "./ArtworkRecommendationsApp"
+    ),
+  {
+    resolveComponent: component =>
+      component.ArtworkRecommendationsAppFragmentContainer,
+  },
+)
+
+export const artworkRecommendationsRoutes: RouteProps[] = [
+  {
+    path: "/artwork-recommendations",
+    getComponent: () => ArtworkRecommendationsApp,
+    onPreloadJS: () => {
+      ArtworkRecommendationsApp.preload()
+    },
+    prepareVariables: (params, props) => {
+      const first = Number.parseInt(props.location.query.first, 10) || 100
```

Comment:
Could we avoid the extra parsing step by setting a default value for `first` in the query?
```
query artworkRecommendationsRoutes_TopLevelQuery($first: Int = 100) {
  me {
    ...ArtworkRecommendationsApp_me @arguments(first: $first)
  }
}
```